Strategy11/formidable-forms
# Make field options grid clickable on mobile(iphone)

Fix https://github.com/Strategy11/formidable-pro/issues/5431
**Note**: Go to Formidable > Styles and click on 'Update' on the styler page to generate a new stylesheet used on your pages before testing.

### Test steps
1. Add a couple of checkbox fields to a form.
2. Create a Grid Layout using the steps outlined [here](https://formidableforms.com/knowledgebase/form-layout-and-css-classes/#kb-create-a-grid)
3. Preview the form using BrowserStack or available iPhone 14, 15, 16 within Chrome.
4. Confirm that the checkboxes are selectable when you tap on them.

![CleanShot 2024-10-22 at 17 43 59](https://github.com/user-attachments/assets/c74fdb3d-8d79-447f-9506-c95a7311d71d)
